src/rust/en.omegascans/res/source.json

```
@@ -3,17 +3,20 @@
 		"id": "en.omegascans",
 		"lang": "en",
 		"name": "Omegascans",
```

looks like this is how they have it on the site
```suggestion
		"name": "Omega Scans",
```